pallets/ddc-payouts/src/lib.rs

```diff
@@ -785,10 +793,14 @@ pub mod pallet {
 						.ok_or(Error::<T>::ArithmeticOverflow)?;
 				}
 
-				Self::deposit_event(Event::<T>::Rewarded {
+				Self::deposit_event(Event::<T>::ProvidersRewarded {
```

Comment:
@yahortsaryk for the new events, do we need to update the go-SDK?

Reply:
I'll update the `network-relayer` and make sure @khssnv is notified